config/application.travis.yml

```diff
@@ -0,0 +1,5 @@
+production:
+  production_url: ""
```

Comment:
You know what never mind. I was thinking of something else entirely. deployment_url is fine. We can have another discussion when we test out the deployment process.